README.md

```diff
@@ -1 +1,8 @@
-# canary-checker
\ No newline at end of file
+# canary-checker
+
+Command to build Docker Image:
```

Comment:
Use markdown formatting e.g.
To build the docker image: `make image`
To run image: `docker run --sysctl net.ipv4.ping_group_range="0   2147483647" -it flanksource/canary-checker run`

The build/build.sh isn't really necessary - it can just be moved to the Makefile

Reply:
build.sh removed, respective information added in README.md.